src/Numerics/ODESolvers/StrongStabilityPreservingRungeKuttaMethod.jl

```diff
@@ -149,6 +149,84 @@ end
     end
 end
 
+"""
+    SSPRK22Heuns(f, Q; dt, t0 = 0)
+
+This function returns a [`StrongStabilityPreservingRungeKutta`](@ref) time stepping object
+for explicitly time stepping the differential
+equation given by the right-hand-side function `f` with the state `Q`, i.e.,
+
+```math
+  \\dot{Q} = f(Q, t)
+```
+
+with the required time step size `dt` and optional initial time `t0`.  This
+time stepping object is intended to be passed to the `solve!` command.
+
+This uses the second-order, 2-stage, strong-stability-preserving, Runge--Kutta scheme
+of Shu and Osher (1988) (also known as Heun's method.)
+Exact choice of coefficients from wikipedia page for Heun's method :)
+
+### References
+    @article{shu1988efficient,
+      title={Efficient implementation of essentially non-oscillatory shock-capturing schemes},
+      author={Shu, Chi-Wang and Osher, Stanley},
+      journal={Journal of computational physics},
+      volume={77},
+      number={2},
+      pages={439--471},
+      year={1988},
+      publisher={Elsevier}
+    }
```

Comment:
I'm a fan of included the original reference too

```suggestion
    }
    @article {Heun1900,
       title = {Neue Methoden zur approximativen Integration der
       Differentialgleichungen einer unabh\"{a}ngigen Ver\"{a}nderlichen}
       author = {Heun, Karl},
       journal = {Z. Math. Phys},
       volume = {45},
       pages = {23--38},
       year = {1900}
    }
```